Clients/GUI/Shared/MainLayout.razor

```diff
@@ -10,4 +11,14 @@
     </div>
 </div>
 
+<div class="newsbar sidebar">
+    <div class="newsbar-item currencies text-center">
+        <CurrencyBar Codes="codes" />
+    </div>
+</div>
+
 <BlazoredModal />
+
+@code{
+    List<string> codes = new List<string> { "USD", "EUR" };
```

Comment:
Not sure here, but can we make it `var`?

Reply:
"the contextual keyword var may only appear within a local variable declaration or in script code"